lib/DDG/Spice/DebVersion.pm

```diff
@@ -0,0 +1,24 @@
+# Get the version of a package in various debian repositories
+package DDG::Spice::DebVersion;
+
+use Text::Trim;
+use DDG::Spice;
+
+triggers startend => "debian version", "debian versions", "deb ver", "debian ver";
+
+# spice from => '([^/]+)/?(?:([^/]+)/?(?:([^/]+)|)|)';
+spice to => 'http://sources.debian.net/api/src/$1/';
+
+spice wrap_string_callback => 1;
+
+handle remainder => sub {
+    my $query = lc trim $_;
```

Comment:
Nitpicking but probably cleaner to use `handle remainder_lc` so that `$_` is already lowercased for you.